### ISSUE TITLE
DSpace 6.x REST API /registries/schema endpoint incorrectly handles fields without qualifiers

### ISSUE THREAD

**Issue by alanorth** [DSpace/DSpace#7946]
**Describe the bug**
The [DSpace 6.x REST API](https://wiki.lyrasis.org/display/DSDOC6x/REST+API) has an endpoint that allows you to get information about metadata fields in the registry. DSpace does not handle metadata fields without qualifiers properly, for example `dc.title` or `dc.contributor`, and returns HTTP 500 to the user. It logs the following to `dspace.log`:

```console
ERROR org.dspace.rest.MetadataRegistryResource @ Field dc.title. not found
```

These fields obviously exist, and they are visible in the `dc` schema registry: https://demo.dspace.org/rest/registries/schema/dc

~~This probably affects DSpace 4 and 5 as well, since the REST API is the same code.~~ The registries endpoint was added in DSpace 6.

**To Reproduce**
Make a request to the `/rest/registries/schema/` endpoint for a metadata field without a qualifier, for example:

- https://demo.dspace.org/rest/registries/schema/dc/metadata-fields/title
- https://demo.dspace.org/rest/registries/schema/dc/metadata-fields/contributor

**Expected behavior**
An HTTP 200 response with information about the metadata field, for example `dc.contributor.advisor`:

- https://demo.dspace.org/rest/registries/schema/dc/metadata-fields/contributor/advisor

**Comment by ammanabrolua** [comment]
Hi @tdonohue . I noticed one thing. In the org.dspace.content.dao.impl.MetadataFieldDAOImpl in line number - 48

`                    "AND qualifier = :qualifier");
`  
this must be `ms.qualifier` or `mf.qualifier` ? Correct me if I am wrong.

**Comment by tdonohue** [comment]
Closing as this appears to have been fixed by #7993
